dashboard/app/models/ability.rb

```diff
@@ -95,6 +95,9 @@ def initialize(user)
       can :create, Activity, user_id: user.id
       can :create, UserLevel, user_id: user.id
       can :update, UserLevel, user_id: user.id
+      can :create, UserLevelInteraction do
+        user.student?
+      end
```

Comment:
Do we also want to not create ULIs for non-CSP levels (at least for the time being) and if so, where's the best place to do that?